docs/system-requirements/software/perl.rst

```diff
@@ -48,7 +48,7 @@ To change a setting manually, type:
   $ o conf [SETTING NAME] "NEW VALUE"
 
 
-eg to make modules installed uninstall previous versions and use sudo to elevate permissions (very good ideas), type:
+example, to make installed modules uninstall previous versions and use sudo to elevate permissions (very good ideas), type:
 
```

Comment:
I think this is better
for example, to uninstall previous versions of the installed modules and ...